packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -316,7 +345,7 @@ known, it can be explicitly provided to attach via the command-line, e.g.
         ).asBroadcastStream();
     }
 
-    globals.terminal.usesTerminalUi = daemon == null;
+    _terminal.usesTerminalUi = daemon == null;
```

Comment:
Seems strange to me that we set this here; I would think that we would do it somewhere earlier. I'm just talking out loud though, keeping status quo LGTM for now.